src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp

```diff
@@ -1241,6 +1238,10 @@ FixedwingPositionControl::control_position(const math::Vector<2> &current_positi
 			_att_sp.roll_body = _l1_control.nav_roll();
```

Comment:
Yes, but then you would have code duplication since you also need the assignment in the if part.